public/js/interface.js

```diff
@@ -0,0 +1,119 @@
+var validate = function(item, quantity, price) {
+  var count = $("#tillNumbers").children().length;
+  if (quantity !== 0 && count < 7) {
+    $.post("/items", {item: item, quantity: quantity}, function(data) { 
+      addToList();
+    });
+  }
+  else if (quantity === 0){
+    $("#errorTill").text("Enter a Quantity");
+  }
+  else {
+    $("#errorTill").text("We can't fill anymore orders");
+  }
+};
+
+var addToList = function(){
+  $.get("/items", function(data) {
+     calculateTotal();
+     $("#errorTill").empty();  
+     $("#tillNumbers").empty();
+    var size = +data.item.length;
+    for (var i = 0; i < size; i++) {
+      button = data.item[i] + " x" + data.quantity[i] + "= £" + (+data.price[i]).toFixed(2)
+       + " " + '<button value="'+ data.item[i]
+       +'" class="x '+ data.item[i].replace(/\s+/g, '') +'" id="'+ data.quantity[i]
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Bad line breaking before '+'.